### PR TITLE
docs: add DOI and publication

### PR DESCRIPTION
**Pull request by mirestrepo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 # Honeycomb
 
-[![DOI](https://zenodo.org/badge/201963539.svg)](https://zenodo.org/badge/latestdoi/201963539) [![docs](https://img.shields.io/badge/docs-stable-blue)](https://brown-ccv.github.io/honeycomb-docs/)
+[![DOI:10.1590/1516-4446-2020-1675](https://img.shields.io/badge/DOI-10.1590%2F1516--4446--2020--1675-orange)](https://doi.org/10.1590/1516-4446-2020-1675) [![docs](https://img.shields.io/badge/docs-stable-blue)](https://brown-ccv.github.io/honeycomb-docs/)
 
 Honeycomb is a template for reproducible psychophysiological tasks for clinic, laboratory, and home use. It is maintained by members of the [Center for Computation and Visualization](https://ccv.brown.edu) and the [Neuromotion Lab](http://borton.engin.brown.edu/) at Brown University. To learn about installation, usage and deployment please [visit our documentation](https://brown-ccv.github.io/honeycomb-docs/).
 
 *Note: previously named Neuro Task Starter, some references may still refer to it as such*
+
+If you use Honeycomb in your work, please cite
```
Comment (gtdang) [inline README.md]: Consider adding ":" after sentence.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,11 @@
 # Honeycomb
 
-[![DOI](https://zenodo.org/badge/201963539.svg)](https://zenodo.org/badge/latestdoi/201963539) [![docs](https://img.shields.io/badge/docs-stable-blue)](https://brown-ccv.github.io/honeycomb-docs/)
+[![DOI:10.1590/1516-4446-2020-1675](https://img.shields.io/badge/DOI-10.1590%2F1516--4446--2020--1675-orange)](https://doi.org/10.1590/1516-4446-2020-1675) [![docs](https://img.shields.io/badge/docs-stable-blue)](https://brown-ccv.github.io/honeycomb-docs/)
 
 Honeycomb is a template for reproducible psychophysiological tasks for clinic, laboratory, and home use. It is maintained by members of the [Center for Computation and Visualization](https://ccv.brown.edu) and the [Neuromotion Lab](http://borton.engin.brown.edu/) at Brown University. To learn about installation, usage and deployment please [visit our documentation](https://brown-ccv.github.io/honeycomb-docs/).
 
 *Note: previously named Neuro Task Starter, some references may still refer to it as such*
```
Comment (gtdang) [inline README.md]: Just some capitalization and punctuation. 
*Note: Previously named Neuro Task Starter, some references may still refer to it as such.*

**Review by gtdang — APPROVED** [review]
Tested the links and they all work. Added some minor formatting suggestions.

File: README.md (hunk 2 above)
Comment (mirestrepo) [inline README.md]:
```suggestion
*Note: previously named Neuro Task Starter, some references may still refer to it as such.*
```

**Review by RobertGemmaJr — APPROVED** [review]
🚀

File: README.md (hunk 1 above)
Comment (mirestrepo) [inline README.md]:
```suggestion
If you use Honeycomb in your work, please cite:
```

**Pull request merged**